navigation/AppNavigator.js

```diff
@@ -1,99 +1,63 @@
-  render() {
-    return null;
-  }
+import AuthStackNavigator from './stack_navigators/AuthStack';
+import ResourcesStackNavigator from './stack_navigators/ResourcesStack';
+import StoresStackNavigator from './stack_navigators/StoresStack';
+
+const Drawer = createDrawerNavigator();
+
+function DrawerNavigator() {
+  return (
+    <Drawer.Navigator
+      drawerContent={props => <DrawerContent {...props} />}
+      drawerStyle={{ width: 189 }}
+      drawerContentOptions={{
+        labelStyle: {
+          fontFamily: 'poppins-medium',
+          fontSize: 20,
+        },
+        activeTintColor: Colors.primaryGreen,
+      }}>
+      <Drawer.Screen
+        name="Stores"
+        component={StoresStackNavigator}
+        options={{ title: 'Stores' }}
+      />
+      <Drawer.Screen
+        name="Rewards"
+        options={{ title: 'Points History', drawerLockMode: 'locked-closed' }}>
+        {props => <RewardsScreen {...props} tab={1} />}
+      </Drawer.Screen>
+      <Drawer.Screen
+        name="Resources"
+        component={ResourcesStackNavigator}
+        options={{ title: 'Resources' }}
+      />
+    </Drawer.Navigator>
+  );
 }
 
-const MyDrawerNavigator = createDrawerNavigator(
-  {
-    Root: {
-      screen: RootStack,
-      navigationOptions: () => ({
-        title: 'Root',
-        drawerLabel: () => null,
-      }),
-    },
-    Stores: {
-      screen: StoresStack,
-      navigationOptions: () => ({
-        title: 'Stores',
-      }),
-    },
-    Rewards: {
-      screen: props => <RewardsScreen {...props} tab={1} />,
-      navigationOptions: () => ({
-        title: 'Points History',
-        drawerLockMode: 'locked-closed',
-      }),
-    },
-    Resources: {
-      screen: ResourcesStack,
-      navigationOptions: () => ({
-        title: 'Resources',
-      }),
-    },
-  },
-
-  {
-    contentOptions: {
-      labelStyle: {
-        fontFamily: 'poppins-medium',
-        fontSize: 20,
-      },
-      activeTintColor: Colors.primaryGreen,
-    },
-    drawerWidth: 189,
-    contentComponent: DrawerContent,
-  }
-);
-
-export default createAppContainer(
-  createSwitchNavigator(
-    // You could add another route here for authentication.
-    // Read more at https://reactnavigation.org/docs/en/auth-flow.html
-    {
-      AuthLoading: AuthLoadingScreen,
-      App: {
-        screen: MyDrawerNavigator,
-      },
-      Auth: AuthStack,
-    },
-    {
-      initialRouteName: 'AuthLoading',
-    }
-  )
-);
+const AppStack = createStackNavigator();
+
+export default function createAppContainer() {
+  return (
+    <NavigationContainer>
+      <AppStack.Navigator
+        initialRouteName="AuthLoading"
+        screenOptions={{
+          headerShown: false,
+          cardStyle: { backgroundColor: Colors.lightest },
+        }}>
+        <AppStack.Screen name="AuthLoading" component={AuthLoadingScreen} />
+        <AppStack.Screen name="App" component={DrawerNavigator} />
```

Comment:
When this boots up, you can see it transition/move left from a blank screen to the map. Can you disable animations here to get rid of that?